### ISSUE TITLE
Add COMPADRE databases

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#891]
http://www.compadre-db.org/

**Comment by kvnamipara** [comment]
@ethanwhite I will work on this. Thanks.

**Comment by kvnamipara** [comment]
@ethanwhite 
The above database provide data in form of `.RData` file. How to proceed with this one?
Thanks.

**Comment by ethanwhite** [comment]
Ah, that is a problem. Let me see if I know someone in that group who would get a csv version posted for us.

**Comment by ethanwhite** [comment]
I have a meeting tomorrow with the COMPADRE folks to talk more about this.

**Comment by kvnamipara** [comment]
@ethanwhite Great!! Hope we will find out some alternative. Sorry I was inactive for a while, I am busy with my university exams. I will try to contribute from 2nd week of May. Thanks.

**Comment by kvnamipara** [comment]
@ethanwhite have you find any alternatives for this? thanks.

**Comment by ethanwhite** [comment]
@kvnamipara - it turns out to be a little more complicated than I'd assumed. They are using the `RData` format because they've stored the data in what is basically a heterogeneous nested list in which one of the sublists contains thousands of matrices. This makes distributing the data and using it in other languages tricky

I discussed this with them a couple of weeks ago and they said they would like to make it available in another form if we could figure out a way to make that effective. I proposed the following normalized structure after that meeting: https://github.com/jonesor/compadreDB/issues/8, but I haven't heard back yet. I think at this point we're basically waiting on them to respond unless we want to submit a PR to their repo with some R code for transforming their object into my proposed format and the current version of the transformed data.
